services/brig/src/Brig/API/Error.hs

```diff
@@ -37,21 +37,31 @@ import Network.HTTP.Types.Header
 import Network.HTTP.Types.Status
 import qualified Network.Wai.Utilities.Error as Wai
 import Servant.API.Status
-import qualified Wire.API.ErrorDescription as ErrDesc
+import Wire.API.ErrorDescription
 import Wire.API.Federation.Client (FederationError (..))
 import Wire.API.Federation.Error
 
 errorDescriptionToWai ::
   forall (code :: Nat) (lbl :: Symbol) (desc :: Symbol).
   (KnownStatus code, KnownSymbol lbl) =>
-  ErrDesc.ErrorDescription code lbl desc ->
+  ErrorDescription code lbl desc ->
   Wai.Error
-errorDescriptionToWai (ErrDesc.ErrorDescription msg) =
+errorDescriptionToWai (ErrorDescription msg) =
```

Comment:
should this be hidden from export?  we don't want to use it any more outside of this module, right?

Reply:
Some errors are parametrised, and in that case we need to use this function to convert them to `Wai.Error`, because `mkErrorDescription` is not sufficient.